test/coverage/CMakeLists.txt

```diff
@@ -41,10 +41,18 @@ set(COVERAGE_REMOVE_PATHS
     '/usr/*'
     '${PROJECT_SOURCE_DIR}/third-party/*'
     '${PROJECT_SOURCE_DIR}/src/metadata/radixtree/*'
```

Comment:
These files arent there anymore. Can you remove it

    '${PROJECT_SOURCE_DIR}/src/metadata/radixtree/*'